### DISCUSSION TITLE
Debug build process

### DISCUSSION THREAD

**Discussion by hatsumatsu** [vercel/next.js#22186] [Help]
Im wondering what's the best way to debug the build process of a nextJS project?

Context: I'm currently working on a project using ISG and dynamic routes. When running the app in production mode pages are build on the server-side when they are first hit as expected. But it takes ~45s for each page to be build/to load which seems quite a lot to me. How can I check for potential bottlenecks in my code and see which part of the build takes how long?

Using next@10.0.6

**Comment by landsman** [comment]
I just looking for the same thing in similar version.
Did you solved it?
